doc/dev/mountpoints.md

```diff
@@ -32,7 +32,13 @@ The _mountpoint config_ is defined by the keys below `system:/elektra/mountpoint
 It is merged into the configuration keyset of every plugin of this mountpoint (including the backend plugin).
 Specifically, the mountpoint config is put into the `system:/` namespace, while the config from `system:/elektra/mountpoints/<mountpoint>/plugins/<ref>/config` is put into the `user:/` namespace of the keyset passed to a plugin.
 
-<!-- TODO: cover config/needs dynamic config; put into default:/ at runtime ? -->
+<!-- TODO: implement `config/needs` -->
```

Comment:
Currently, `config/needs` is read during `kdb mount` and into the mountpoint configuration and persisted. That means, if a plugin receives an update and changes the `config/needs` in it's contract, the changes will only apply for newly created mountpoints.

Reply:
Good point. I created #4494 to tackle this (amongst other) issues.

Reply:
Actually, I would tackle `config/needs` differently. It is simple enough to always load it at runtime. We don't need to remember the command for that. See comment below.

Reply:
Yes, that would be one problem less. But as said above: the docu should reflect what is, not what could be done. So if you also implement this before merging, I am a big fan of it :+1:

Reply:
Probably not before merging, but before release. I'll explain what should happen, in case someone else wants to do it before I have time, and also for future reference for myself. But I'll clearly mark it as "not implemented yet".